### DISCUSSION TITLE
Moving from Kirby and Notion to Quarto

### DISCUSSION THREAD

**Discussion by neeldhara** [quarto-dev/quarto-cli#3480] [Show and Tell]
I used to have my website on [Kirby](https://getkirby.com), and then moved on to using Notion for content and NextJS to render the HTML. However, I've recently moved away from Notion as my "everything tool", and I always had a hard time remembering what was what in Kirby (since I'd thrown in one customization too many 👀). 

I am so glad I discovered Quarto: setting things up was a lot of fun. I think the main customisation I did was with the backgrounds, fonts, colors, and layouts. [Here](https://www.neeldhara.com) is what I have so far.

There's still a few things I am figuring out:

- two-level nesting on the main navigation (is this easily doable?)
- keeping the backgrounds fixed on mobile
- cleaning up the console errors on the [talks](https://www.neeldhara.com/talks) page
- incorporating a bib-to-yaml script so I can do nicer tables on the publications page with a ejs template (not sure if the ejs templates can directly pull from bib files)
- embedding a mastodon feed somewhere

Of course, a lot of content is currently `todo`, but I just wanted to give an excited shout out and thanks at this intermediate stage 😀

**Comment by camilogarciabotero** [comment]
Hey, your bookmarks list is awesome!

**Comment by baptiste** [comment]
Super good website!

**Comment by mcanouil** [comment]
Checkout my website for the "bib to yaml", this can get you started to achieve what you want.

<https://github.com/mcanouil/mickael.canouil.fr/blob/main/publications.R> (as a pre-render script)
